invenio_app_rdm/theme/templates/semantic-ui/invenio_theme/header_login.html

```diff
@@ -0,0 +1,23 @@
+{# -*- coding: utf-8 -*-
```

Comment:
this file will not be used if placed here in invenio_app_rdm becuase the application is using the one above, you should move it to invenio_administration instead (so that invenio_administration can be installed in other application and have the menu added out of the box, without overwriting the header)